scripts/gh_scripts/weekly_status_report.mjs

```diff
@@ -234,6 +237,49 @@ async function prepareUntriagedIssues(leads) {
     return output
 }
 
+/**
+ * Finds all issues with the "Needs: Review Assignee" label, prepares a message containing a
+ * summary for each given lead, and returns the prepared messages
+ *
+ * @param {Array<Lead>} leads
+ * @returns {Promise<Array<string>>}
+ */
+async function prepareReviewAssigneeIssues(leads) {
+    const output = ['*Issues needing attention (__Assignee may have abandoned or may need help__)*:']
+
+    const staleIssues = await octokit.paginate('GET /repos/{owner}/{repo}/issues', {
+        owner: 'internetarchive',
+        repo: 'openlibrary',
+        labels: 'Needs: Review Assignee',
+        per_page: 100,
+        headers: {
+          'X-GitHub-Api-Version': '2022-11-28'
+        }
+      })
+
+    let noStaleIssuesFound = true
+    for (const lead of leads) {
+        const issuesForLead = staleIssues.filter((issue) => {
+            for (const label of issue.labels) {
+                if (label.name === lead.leadLabel) {
+                    return true
+                }
+            }
+            return false
+        })
+        const searchResultsUrl = `https://github.com/internetarchive/openlibrary/issues?q=is%3Aopen+is%3Aissue+label%3A%22Needs%3A+Review+Assignee%22+label%3A${encodeURIComponent('"' + lead.leadLabel + '"')}`
+        if (issuesForLead.length > 0) {
+            output.push(`  • <${searchResultsUrl}|${issuesForLead.length} issue(s)> need follow-up from ${lead.slackId}`)
```

Comment:
Since recent comments is also something that requires follow-up from lead, I wonder if we want to label this as:

e.g. 5 issues need to be checked for inactive assignee

Reply:
I actually see that they are batched as labels, I like that -- keep as is :) nice thinking